cisco_aci/datadog_checks/cisco_aci/api.py

```diff
@@ -84,7 +84,8 @@ def make_request(self, path):
         try:
             response.raise_for_status()
         except Exception as e:
-            self.log.warning("Error making request: {}".format(e))
+            self.log.warning("Error making request: %s", e)
+            self.log.debug("Error response content: %s", response.content)
```

Comment:
Since it comes from the same exception, why isn't it a warning?

Reply:
Call to the api seems to be expected to fail from time to time, I suppose that's why it's a warning in the first place. So it might be a bit verbose to log the payload, but agree that it might be helpful if the customer don't have to set log level to DEBUG.

The error response.content payload is not that large. I guess, it's ok to output the full content as warning.

`{"totalCount":"1","imdata":[{"error":{"attributes":{"code":"1","text":"the messaging layer was unable to deliver the stimulus (destination (node) is marked unavailable)"}}}]}`

I'm combining the two logs into one then.